### PR TITLE
Format check alignment 

### PR DESCRIPTION
**Pull request by AzuraTarmiziIntel**
To run : make format V=1 

There is other files to changes still

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -453,8 +453,14 @@ CPPCHECK_OPT+=--suppress=preprocessorErrorDirective
 EXTRA_C_SRCS:=$(wildcard uctest/*.c)
 EXTRA_SRCS:=$(wildcard $(foreach n,sample utest uctest,$n/*.cpp $n/*.h))
 EXTRA_SRCS+=$(EXTRA_C_SRCS)
+
+EXTRA_SRCS+=$(wildcard jclklib/common/*.c*)
+EXTRA_SRCS+=$(wildcard jclklib/proxy/*.c*)
+EXTRA_SRCS+=$(wildcard jclklib/client/*.c*)
+EXTRA_SRCS+=$(wildcard jclklib/sample/*.c*)
```
Comment (yoongsiang2) [inline Makefile:460]: i think no need to add sample, as sample should not compile together, like checksync app

Comment (AzuraTarmiziIntel) [reply Makefile]: this is just for format checking. i think erez do check the format for his sample. so i just follow ? 
EXTRA_SRCS:=$(wildcard $(foreach n,sample utest uctest,$n/*.cpp $n/*.h))
RCS)

Comment (yoongsiang2) [reply Makefile]: oh i see. only for format check.

**Comment by AzuraTarmiziIntel** [comment]
This is for format checking.

**Pull request merged**